### ISSUE TITLE
Add some kind of random number generator

### ISSUE THREAD

**Issue by xMGZx** [Stevertus/mcscript#35]
In basic vanilla functions, random numbers are annoying to generate. This would be a very useful tool in making the process simpler.

**Comment by Heath123** [comment]
Sadly this isn't really supported or updated any more...

**Comment by Stevertus** [comment]
Yeah I think that would be a great usecase for this language, but over time it got hard maintaining this project. This is why I rewrote everything to favor a modular approach and community plugins with objD. Here you can also find a sophisticated random generator: https://objd.stevertus.com/utils/#randomscore
